remove eslint complexity checks

Running npm test on master produces this chirp. I don't think the linter complexity tests have ever uncovered  any real bugs, and mostly just whine about `||` defaulting. Can we remove the linting for "complexity"?

```
Running "eslint:files" (eslint) task

lib/db/mem.js
  306:34  warning  Function 'anonymous' has a complexity of 7  complexity

✖ 1 problem (0 errors, 1 warning)
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -1,5 +1,6 @@
 extends: fxa/server
 
 rules:
+  complexity: 0
   handle-callback-err: 0
   semi: [2, "never"]

--- a/lib/db/mem.js
+++ b/lib/db/mem.js
@@ -304,7 +304,6 @@ module.exports = function (log, error) {
   // Takes:
   //   - id - a string of hex chars
   Memory.prototype.sessionToken = function (id) {
-    /*eslint complexity:7*/
     id = id.toString('hex')
 
     if ( !sessionTokens[id] ) {

